src/ex4/server/helper/logger.js

```diff
@@ -1,11 +1,14 @@
-import { writeToFile, readFile } from './fileHandler.js'
+import { updateFile } from './fileHandler.js'
 
 export const logger = async (req, res, next) => {
-    const currentDate = new Date().toJSON().slice(0, 10).replace(/-/g, '/');
-    let logs = await readFile("./server/data/log.json")
+    const currentDate = new Date().toLocaleDateString()
     if (req.body.item !== undefined) {
-        logs = [...logs, `Item: ${req.body.item}, Date : ${currentDate}`]
-        writeToFile("./server/data/log.json", logs)
+        updateFile("./server/data/log.json", `Item: ${req.body.item}, Date : ${currentDate}`)
     }
+    // let logs = await readFile("./server/data/log.json")
+    // if (req.body.item !== undefined) {
+    //     logs = [...logs, `Item: ${req.body.item}, Date : ${currentDate}`]
+    //     writeToFile("./server/data/log.json", logs)
+    // }
```

Comment:
don't leave unnecessary comments in the code - you can move them to a different branch or stash them if you will need them for later